modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -43,7 +43,8 @@
 from pandas.core.indexing import check_bool_indexer
 from pandas.errors import DataError, MergeError
 
-from modin.config import CpuCount, RangePartitioningGroupby
+import modin.pandas as pd
```

Comment:
Why do we need to import this module here? I guess we would need to filter arguments at the API layer and pass QueryCompilers to this layer.

Reply:
The arguments are not filtered - https://github.com/modin-project/modin/blob/9adaf3396cd88b1cc1ab25a72d4373d874cc6069/modin/pandas/series.py#L1018

Reply:
Should I add the filter?

Reply:
Yes, the API layer should filter arguments to pass those in the QC.

Reply:
Added the filtering, but when defaulting to pandas the arguments are not properly converted to Series. Should be fixed in a separate PR.

Reply:
Do you want to first fix this case in a separate PR and only after that complete this PR?

Reply:
I tried to fix in this PR by adding flag to the Series qc and then squeeze when converting to Pandas, but got a lot of test failures. Seems, the fix is a bit more complicated. In this PR I've added a workaround by converting the arguments, when defaulting to pandas, explicitly. We could live it as is and then remove then a proper conversion mechanism is implemented.